kernels/cpu/include/readdy/kernel/cpu/data/ObservableData.h

```diff
@@ -48,20 +48,16 @@
 #include <readdy/model/reactions/ReactionRecord.h>
 #include <readdy/common/common.h>
 
-namespace readdy {
-namespace kernel {
-namespace cpu {
-namespace data {
+namespace readdy::kernel::cpu::data {
 
 struct ObservableData {
     std::vector<readdy::model::reactions::ReactionRecord> reactionRecords;
-    readdy::model::reactions::reaction_counts_map reactionCounts;
+    readdy::model::reactions::ReactionCounts reactionCounts;
```

Comment:
consider default initializing all members in this struct

Reply:
ok!